wix-custom-labels/src/index.js

```diff
@@ -0,0 +1,40 @@
+const isEqual = require('lodash/isEqual')
+const CUSTOM_LABELS_KEY = 'customLabels'
+const CUSTOM_LABELS_TAG = 'custom-labels'
+
+function defineTags(dictionary) {
+  dictionary.defineTag(CUSTOM_LABELS_TAG, {
+      mustNotHaveValue: false,
+      mustHaveValue: true,
+      onTagged: function (doclet, tag) {
+        doclet.customLabels = []
```

Comment:
Not sure which pattern is better:
`@custom-label foo bar`
vs.
`@custom-label foo`
`@custom-label bar`
WDYT?

Reply:
talked with tova from docs team and idan-attias and we are all synced with @customlabels foo bar solution.